pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -128,6 +128,11 @@ type ClusterDeploymentSpec struct {
 	// time since the cluster last came out of hibernation.
 	// +optional
 	HibernateAfter *metav1.Duration `json:"hibernateAfter,omitempty"`
+
+	// InstallAttemptsLimit is the number of times Hive attempts an installation. If the last attempt
+	// fails ClusterDeployment will show an error in the status
```

Comment:
The second sentence of this description does not add very much. I suggest that it either be removed or expanded to describe the type of error and where in the status the error would appear. Is the error different than the normal error that would show up if `InstallAttemptsLimit` were omitted?

Reply:
Keeping it short sounds better. Removing the second sentence.

Reply:
> Is the error different than the normal error that would show up if `InstallAttemptsLimit` were omitted?

No